mtail-formula/mtail/init.sls

```diff
@@ -0,0 +1,90 @@
+{#-
+Salt state file for managing mtail
+Copyright (C) 2023 Georg Pfuetzenreuter <mail+opensuse@georg-pfuetzenreuter.net>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+-#}
+
+{%- from 'mtail/map.jinja' import config %}
+
+mtail_package:
+  pkg.installed:
+    - name: mtail 
+
+{%- set sysconfig = [] %}
+{%- for key, value in config['sysconfig']['args'].items() %}
+{%- if value is string %}
+{%- do sysconfig.append('-' ~ key ~ ' ' ~ value) %}
+{%- elif value is sameas true %}
+{%- do sysconfig.append('-' ~ key) %} %}
+{%- else %}
+{%- do salt.log.error('mtail: illegal sysconfig value') %}
```

Comment:
I'd recommend to also allow / check for `false` before erroring out

Reply:
Good point!